OpenRA.Game/Network/Order.cs

```diff
@@ -239,21 +252,27 @@ public static Order CancelProduction(Actor subject, string item, int count)
 
 		// For scripting special powers
 		public Order()
-			: this(null, null, Target.Invalid, null, false, CPos.Zero, 0) { }
+			: this(null, null, Target.Invalid, null, false, null, CPos.Zero, 0) { }
 
 		public Order(string orderString, Actor subject, bool queued)
-			: this(orderString, subject, Target.Invalid, null, queued, CPos.Zero, 0) { }
+			: this(orderString, subject, Target.Invalid, null, queued, null, CPos.Zero, 0) { }
 
 		public Order(string orderString, Actor subject, Target target, bool queued)
-			: this(orderString, subject, target, null, queued, CPos.Zero, 0) { }
+			: this(orderString, subject, target, null, queued, null, CPos.Zero, 0) { }
+
+		public Order(string orderString, Actor subject, Target target, bool queued, List<Actor> extraActors)
+			: this(orderString, subject, target, null, queued, extraActors, CPos.Zero, 0) { }
 
 		public byte[] Serialize()
 		{
 			var minLength = 1 + OrderString.Length + 1;
 			if (Type == OrderType.Handshake)
 				minLength += TargetString.Length + 1;
 			else if (Type == OrderType.Fields)
-				minLength += 4 + 1 + 13 + (TargetString != null ? TargetString.Length + 1 : 0) + 4 + 4;
+				minLength += 4 + 1 + 13 + (TargetString != null ? TargetString.Length + 1 : 0) + 4 + 4 + 4;
+
+			if (ExtraActors != null)
+				minLength += ExtraActors.Count() * 4;
```

Comment:
Plain old `Count` should work here, right?

Reply:
Fixed.

Reply:
This is still using the Enumerable method, so hasn't been resolved?

Reply:
Weird. Don't know how that happened.

Fixed.